draft-ietf-trans-rfc6962-bis.md

```diff
@@ -757,41 +757,14 @@ the submitter, and the trust anchor used to verify the chain (even if it was
 omitted from the submission). The log MUST present this chain for auditing upon
 request (see {{get-entries}}). This chain is required to prevent a CA from
 avoiding blame by logging a partial or empty chain.
-
-Each certificate entry in a log MUST include a `X509ChainEntry` structure, and
-each precertificate entry MUST include a `PrecertChainEntryV2` structure:
-
-~~~~~~~~~~~
-    opaque ASN.1Cert<1..2^24-1>;
-
-    struct {
-        ASN.1Cert leaf_certificate;
-        ASN.1Cert certificate_chain<0..2^24-1>;
-    } X509ChainEntry;
-
-    opaque CMSPrecert<1..2^24-1>;
-
-    struct {
-        CMSPrecert pre_certificate;
-        ASN.1Cert precertificate_chain<1..2^24-1>;
-    } PrecertChainEntryV2;
-~~~~~~~~~~~
-
-`leaf_certificate` is a submitted certificate that has been accepted by the log.
-
-`certificate_chain` is a vector of 0 or more additional certificates required to
-verify `leaf_certificate`. The first certificate MUST certify
-`leaf_certificate`. Each following certificate MUST directly certify the one
-preceding it. The final certificate MUST be a trust anchor accepted by the log.
-If `leaf_certificate` is an accepted trust anchor, then this vector is empty.
-
-`pre_certificate` is a submitted precertificate that has been accepted by the
-log.
-
-`precertificate_chain` is a vector of 1 or more additional certificates required
-to verify `pre_certificate`. The first certificate MUST certify
-`pre_certificate`. Each following certificate MUST directly certify the one
-preceding it. The final certificate MUST be a trust anchor accepted by the log.
+Log entries, which are the input to the Merkle Tree, are TLS-encoded
+`TransItem`s of type `x509_entry_v2` or `precert_entry_v2`.
+To be able to construct a `TransItem` representing the entry in the future, in
+addition to storing the original submission, the log also has to store the
+timestamp of the issued SCT, as well as any SCT extensions.
+Implementations may store all of this information in any format, but MUST
+be able to construct from it the corresponding log entries, in the order they
+were created and added to the tree.
```

Comment:
I suggest deleting this sentence, and then using the following text (note: this incorporates my other wordsmithing attempts during this review)...

"Each log entry is a `TransItem` structure of type `x509_entry_v2` or `precert_entry_v2`. However, a log may store its entries in any format. If a log does not store each `TransItem` in full, it must at least store the `timestamp` and `sct_extensions` of the corresponding `TimestampedCertificateEntryDataV2` structure. Armed with these fields and the entire chain used to verify each submission, a log will be able to reconstruct each `TransItem`."